cpp/models/ode_secir/parameter_space.cpp

```diff
@@ -115,17 +115,15 @@ void draw_sample_infection(Model& model)
     model.parameters.get<Seasonality>().draw_sample();
 
     //not age dependent
-    model.parameters.get<IncubationTime>()[AgeGroup(0)].draw_sample();
-    model.parameters.get<SerialInterval>()[AgeGroup(0)].draw_sample();
+    model.parameters.get<TimeExposed>()[AgeGroup(0)].draw_sample();
```

Comment:
This actually is a mess. For a future version I have always hoped to include some generic (user-defined?) structure which allows to set if age-dependent or not. The problem if age-dependency is that it can be sampled independently and we get different results. For backward stability, we should at the moment keep it in the non age-dependent as before.